tests/integration-tests/tests/cloudwatch_logging/test_compute_console_output_logging/test_custom_action_error/pcluster.config.yaml

```diff
@@ -0,0 +1,60 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Iam:
+    AdditionalIamPolicies:
+      - Policy: arn:aws:iam::aws:policy/AmazonSSMFullAccess
+      - Policy: arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess
+  Ssh:
+    KeyName: {{ key_name }}
+  Imds:
+    Secured: {{ imds_secured }}
+Scheduling:
+  Scheduler: {{ scheduler }}
+  SlurmQueues:
+    - Name: compute-a
+      ComputeResources:
+        - Name: cit-0
+          Instances:
+            - InstanceType: {{ instance }}
+          MinCount: 1
+          MaxCount: 10
+      Networking:
+        SubnetIds:
+          - {{ private_subnet_id }}
+      CustomActions:
+        OnNodeStart:
+          Script: s3://{{ bucket }}/{{ script_path }}
+          Args:
+            - arg1
+            - arg2
+      Iam:
+        AdditionalIamPolicies:
+          - Policy: arn:aws:iam::aws:policy/AmazonSSMFullAccess
```

Comment:
Is SSM Full access required for this test?
Same comment for other configs

Reply:
It is not. Removed from configs.